src/flamenco/runtime/fd_runtime.c

```diff
@@ -1066,6 +1067,12 @@ fd_txn_prep_and_exec_task( void  *tpool,
   fd_execute_txn_task_info_t * task_info = (fd_execute_txn_task_info_t *)tpool + m0;
   fd_exec_slot_ctx_t * slot_ctx = (fd_exec_slot_ctx_t *)args;
   // fd_capture_ctx_t * capture_ctx = (fd_capture_ctx_t *)reduce;
+    
+  ulong tile_idx = fd_tile_idx();
```

Comment:
could we explain that there is a 1-1 mapping between the tiles and the tpool threads - there are no tiles that should not be "owned" by a tpool thread